doc/source/dev/dev_quickstart.rst

```diff
@@ -84,24 +85,20 @@ Next, set up your development environment.
         Your command prompt now lists the name of your new environment, like so
         ``(scipy-dev)$``.
 
-Finally, build SciPy for development. On Linux and OSX, you should use::
+Finally, build SciPy for development with::
 
     python dev.py build
 
-If you run into a build issue, or need more detailed build documentation
-including building on Windows, see :ref:`building`.
```

Comment:
The "With [system-level dependencies](https://output.circle-artifacts.com/output/job/84fc2e60-7fad-4280-b709-3f164d02db9a/artifacts/0/html/building/index.html#system-level) installed" bit only appears in the Virtual env tab. I think it is needed in the Conda env tab, too (for Windows at least).

Reply:
Made the text here identical now, and because of that could move it outside the tabs now. Resolving.